StackExchange.Redis/StackExchange/Redis/IDatabase.cs

```diff
@@ -45,6 +45,44 @@ public interface IDatabase : IRedis, IDatabaseAsync
         RedisValue DebugObject(RedisKey key, CommandFlags flags = CommandFlags.None);
 
         /// <summary>
+        /// Add the specified member to the set stored at key. Specified members that are already a member of this set are ignored. If key does not exist, a new set is created before adding the specified members.
+        /// </summary>
+        /// <returns>True if the specified member was not already present in the set, else False</returns>
+        /// <remarks>http://redis.io/commands/sadd</remarks>
+        bool GeoAdd(RedisKey key, RedisValue value0, RedisValue value1, RedisValue value2, CommandFlags flags = CommandFlags.None);

```

Comment:
In the case of `SortedSetAdd`, only the `member` (i.e. the location name in the case of `Geo`) is a `RedisValue`; the `score` is a `double`:

    public bool SortedSetAdd(RedisKey key, RedisValue member, double score,  ...

In the case of `Geo`, the `longitude` and `latitude` and kinda like the `score`, so they should a: be `double` and b: be named clearly.